ideascube/serveradmin/catalog.py

```diff
@@ -394,11 +394,12 @@ def install(self, download_path, install_dir):
         for media in manifest['medias']:
             try:
                 self._install_media(media, pseudo_install_dir)
-            except:
+            except Exception as e:
                 # This can lead to installed package with uninstall media.
                 # We sould handle this somehow.
                 printerr("Cannot install media {} from package {}".format(
                     media['title'], self.id))
+                printerr(e)
                 continue
 
     def _install_media(self, media_info, pseudo_install_dir):
```

Comment:
Hahaha, I just realized we both did the same thing: https://github.com/ideascube/ideascube/pull/733/commits/f6cac3da3c4048559b97dfea1ba92c7fa22d7154

😆 

I sort of prefer having the error on the same line as the "Cannot install..." summary, otherwise it's not entirely clear to the user that they are related.

I'm happy to remove it from my pull request since you had done it first.